code/GraphQL/Resolvers/AssetAdminResolver.php

```diff
@@ -217,6 +218,34 @@ public static function resolvePublicationResultUnion($value): string
         }
     }
 
+    public static function resolveReadDescendantFileCounts($object, array $args, $context, ResolveInfo $info): array
+    {
+        /** @var DataList|File[] $files */
+        /** @var File|AssetAdminFile $file */
+        if (!is_array($args['ids'] ?? 0)) {
```

Comment:
Is the point of the `??` operator here to squash the "Undefined index" notice?

Seems like a somewhat abusive approach. I rather we did something a little more explicit like this.

```suggestion
        if (!isset($args['ids']) || !is_array($args['ids'])) {
```

Reply:
Updated